meinberlin/react/livequestions/QuestionForm.jsx

```diff
@@ -65,36 +65,45 @@ export default class QuestionForm extends React.Component {
 
   render () {
     return (
-      <form id="id-comment-form" action="" onSubmit={this.addQuestion.bind(this)}>
+      <>
         <h2>{askQuestionStr}</h2>
-        {Object.keys(this.props.category_dict).length > 0
-          ? <CategorySelect
-              onSelect={this.selectCategory.bind(this)}
-              category_dict={this.props.category_dict}
+        <form id="id-comment-form" className="form--base panel--heavy" action="" onSubmit={this.addQuestion.bind(this)}>
```

Comment:
we could add label to the form, maybe w/ aria-labelledby and the h2 content? what do you think?
```
<h2 id="form-heading">{askQuestionStr}</h2>
<form [...] aria-labelledby="form-heading"> 
```

Reply:
Nice, thanks. Added!